libs/langgraph/langgraph/prebuilt/tool_validator.py

```diff
@@ -175,9 +174,7 @@ def __init__(
                         f"Tool {schema.name} does not have an args_schema defined."
                     )
                 self.schemas_by_name[schema.name] = schema.args_schema
-            elif isinstance(schema, type) and issubclass(
```

Comment:
we don't want to remove support for pydantic model v1

Reply:
for tool validation? i thought we wanted v1 and v2 support only for state schema?